python/cuml/test/test_random_forest.py

```diff
@@ -340,51 +358,83 @@ def test_rf_regression_float64(large_reg, datatype):
                                        convert_dtype=False)
 
 
-@pytest.mark.parametrize('datatype', [(np.float32, np.float32)])
-@pytest.mark.parametrize('column_info', [unit_param([20, 10]),
-                         quality_param([200, 100]),
-                         stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(500), quality_param(5000),
-                         stress_param(500000)])
-@pytest.mark.parametrize('n_classes', [10])
-@pytest.mark.parametrize('type', ['dataframe', 'numpy'])
-def test_rf_classification_multi_class(datatype, column_info, nrows,
-                                       n_classes, type):
+def check_predict_proba(test_proba, baseline_proba, y_test, rel_err):
+    y_proba = np.zeros(np.shape(baseline_proba))
+    for count, _class in enumerate(y_test):
+        y_proba[count, _class] = 1
+    baseline_mse = mean_squared_error(y_proba, baseline_proba)
+    test_mse = mean_squared_error(y_proba, test_proba)
+    assert test_mse <= baseline_mse * (1.0 + rel_err)
```

Comment:
please can you add a comment above the assert line as to why did you choose this threshold. This will help in the future when we are trying to debug things